premarum-backend/PreEnrollmentMgmt.WebApi/Controllers/DTOS/Requests/NewPreEnrollmentRequest.cs

```diff
@@ -0,0 +1,9 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace PreEnrollmentMgmt.WebApi.Controllers.DTOS.Requests;
+
+public class NewPreEnrollmentRequest
+{ 
+    [Required] public string Name { get; set; }
```

Comment:
Should the Name have length constraints? In the database the field is a varchar(100)